test/jdk/javax/swing/JFileChooser/FileViewNPETest.java

```diff
@@ -36,7 +36,6 @@
 /*
  * @test
  * @bug 6616245
- * @key headful
```

Comment:
Why did you remove `headful`?

Reply:
Added again, though some manual tests have this tag and some don't better to keep it as an indication that represents as headful test......